docs/06/01/faq.md

```diff
@@ -0,0 +1,177 @@
+## Component Adapter FAQ
+
+This FAQ addresses common questions and potential issues when working with the Component Adapter system in the Gusto Embedded React SDK.
+
+### General Questions
+
+#### How do I know which components I can customize?
+
+You can customize any component defined in the `ComponentsContextType` interface ([View interface on GitHub](https://github.com/Gusto/embedded-react-sdk/blob/main/src/contexts/ComponentAdapter/useComponentContext.ts)). These are all the components in the SDK's UI directory. If you're unsure, you can refer to the [Component Inventory](./component-inventory.md) section.
+
+#### Do I need to implement all components in the adapter?
+
+No, you only need to implement the components you want to customize. The SDK will use its default components for any components not provided in your adapter.
+
+To use a combination of custom and default components, merge your implementations with the `defaultComponents` export ([View on GitHub](https://github.com/Gusto/embedded-react-sdk/blob/main/src/contexts/ComponentAdapter/adapters/defaultComponentAdapter.tsx)):
+
+```tsx
+import { defaultComponents } from '@gusto/embedded-react-sdk'
+
+const myAdapter = {
+  ...defaultComponents,
+  Button: props => <MyCustomButton {...props} />,
+  TextInput: props => <MyCustomTextInput {...props} />,
+}
+```
+
+### Design and Styling
+
+#### What's the difference between using the Component Adapter and just overriding the CSS?
+
+The Component Adapter gives you full control over the implementation of UI components, including their structure, behavior, and styling. Overriding CSS only allows you to modify the appearance of the default components.
+
+The Component Adapter is ideal when:
+
+- You want to maintain visual consistency with your existing React design system
+- You need to use specific UI component libraries that aren't compatible with the SDK's default styling
+- You need to modify the behavior of components beyond what's possible with styling
+
+#### How can I ensure my custom components maintain accessibility features?
+
+When implementing custom components, pay attention to:
+
+- Proper labeling of form controls
+- Appropriate ARIA attributes
+- Keyboard navigation
+- Focus management
+- Color contrast
```

Comment:
I'm wondering if this actually needs to be more of a disclaimer that if they are using the adapter, they will own the accessibility for custom components and pointing them to some generally a11y resources